src/components/searchSelfCareView/ServiceSearch.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Box,
+  FormControl,
+  FormHelperText,
+  OutlinedInput,
+} from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
+import Search from "../../modules/Search";
+
+const ServiceSearch = ({ setSearchResults }) => {
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const performSearch = async () => {
+    const response = await Search.create(searchQuery);
+    setSearchResults(response.data);
+  };
+
+  return (
+    <Box data-cy="self-care-search-bar" style={styles.searchBar}>
+      <FormControl>
+        <OutlinedInput
+          data-cy="search-query"
+          onChange={(e) => setSearchQuery(e.target.value)}
+          color="secondary"
+          placeholder="Search for a service..."
+          aria-describedby="Search for self care services"
```

Comment:
while at it add for example aria-labelledby="Search input" to show describe the essence of the object 👍

Reply:
Seems to be a lot to do with aria to so might be better to add it as a chore and do it asap and then add it continuously through every PR.. I need to read up more on this part aria